database/raft/raft.go

```diff
@@ -442,31 +426,29 @@ func (sv *Service) allocNodeID(ctx context.Context) (uint64, error) {
 		nextID, index = sv.state.NextNodeID()
 		log.Printf(ctx, "raft: attempting to allocate nodeID %d at version %d", nextID, index)
 		sv.stateMu.Unlock()
-		b := state.IncrementNextNodeID(nextID, index)
-		err = sv.exec(ctx, b)
+		b := sv.state.IncrementNextNodeID(nextID, index)
+		err = sv.Exec(ctx, b)
 	}
 	return nextID, err //caller should check for error b/c value of nextID is untrustworthy in that case
 }
 
-// Get gets a value from the key-value store.
-// It is linearizable; that is, if a
-// Set happens before a Get,
-// the Get will observe the effects of the Set.
-// (There is still no guarantee an intervening
-// Set won't have changed the value again, but it is
+// RequestRead requests a linearizable read. Upon successful return,
+// the caller is guaranteed that a read from the Service's state will
+// be linearizable; this is, if a Set happens before a Get, the Get
+// will observe the effects of the Set. (There is still no guarantee
+// an intervening Set won't have changed the value again, but it is
 // guaranteed not to read stale data.)
-// This can be slow; for faster but possibly stale reads, see Stale.
-func (sv *Service) Get(ctx context.Context, key string) ([]byte, error) {
+func (sv *Service) RequestRead(ctx context.Context) error {
 	for {
-		resp, err := sv.get(ctx, key)
+		err := sv.linearizeRead(ctx)
 		if isTimeout(err) {
 			continue
 		}
-		return resp, err
+		return err
 	}
 }
 
-func (sv *Service) get(ctx context.Context, key string) ([]byte, error) {
+func (sv *Service) linearizeRead(ctx context.Context) error {
```

Comment:
I know this is wordier, but could we call this `linearizableRead` instead? `linearizeRead` sounds like we are taking a read and linearizing it, whatever that means.